webapp/src/blocks.tsx

```diff
@@ -693,8 +693,8 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     protected resizeFieldEditorView() {
         if (!window) return;
         const blocklyDiv = this.getBlocksEditorDiv();
-        if (blocklyDiv && this.parent.isTutorial() && !pxt.BrowserUtils.isTabletSize()) {
-            const containerRect = blocklyDiv.getBoundingClientRect();
+        const containerRect = blocklyDiv.getBoundingClientRect();
```

Comment:
It's *probably* safe, but right now theres a bad check on blocklyDiv that reads as unsafe (i.e. would have crashed here here but you have it in the if on the next line still). How about `blocklyDiv?.getBoundingClientRect()` here and check `containerRect` instead of `blocklyDiv` on the next line?